### PR TITLE
Improve typography.

### PR DESCRIPTION
**Pull request by hghwng**
Improve typography in README.md.

Before:

>To go to the general documentation [click here][DOCUMENTATION.org].
>
> You can also check the [Quick Start guide][QUICK_START.org] and the [FAQ][FAQ.org].

After:

> You can also check the [general documentation][DOCUMENTATION.org],  [quick start guide][QUICK_START.org] and the [FAQ][FAQ.org].

Before:
 
> *Note:* after you have completed the [install process below](#install), it is

After:

> *Note:* after you have completed the [install process](#install) below, it is

Before:
 
> Good quality builds can be found [on this page][emacs-for-windows]. It is recommended to install the most stable build.

After:

>You can download good quality builds from the [emacs-w64 project][emacs-for-windows]. It is recommended to install the [most stable build][emacs-for-windows-stable].
 

Before:
> Source: [Stackoverflow][so-server-unsafe]

After:
> Source: [Stack Overflow][so-server-unsafe]

Before:
 
> If you are using Ubuntu and Unity then you can add the Spacemacs logo by following the instructions [here][cpaulik-unity-icon].

After:

> For Ubuntu users, follow this guide to [change the logo in Unity][cpaulik-unity-icon].

Before:
 
> If you're on a mac follow [these instructions][icon-mac-instructions]. You can find an .icns version of the logo by [Nasser Alshammari](http://www.nass3r.com) in [his repository][icon-repository].

After:

> For Mac users, you need to [download the .icns version of the logo][icon-repository], then [change the logo on Dock][icon-mac-instructions].

[Twitter]: http://i.imgur.com/tXSoThF.png
[CONTRIBUTING.org]: CONTRIBUTING.org
[CONVENTIONS.org]: http://spacemacs.org/doc/CONVENTIONS
[DOCUMENTATION.org]: http://spacemacs.org/doc/DOCUMENTATION
[QUICK_START.org]: http://spacemacs.org/doc/QUICK_START
[FAQ.org]: http://spacemacs.org/doc/FAQ
[VIMUSERS.org]: http://spacemacs.org/doc/VIMUSERS
[dotfile]: http://spacemacs.org/doc/DOCUMENTATION#orgheadline45
[osx layer]: http://spacemacs.org/layers/osx/README
[Gitter Chat]: https://gitter.im/syl20bnr/spacemacs
[Gitter Chat IRC server]: https://irc.gitter.im/
[homebrew]: http://brew.sh
[emacs-for-windows]: http://emacsbinw64.sourceforge.net/
[emacs-for-windows-stable]: https://sourceforge.net/projects/emacsbinw64/files/release/
[the platinum searcher]: https://github.com/monochromegane/the_platinum_searcher
[so-server-unsafe]: http://stackoverflow.com/questions/885793/emacs-error-when-calling-server-start
[Spacemacs logo]: https://github.com/nashamri/spacemacs-logo
[Nasser Alshammari]: https://github.com/nashamri
[cpaulik-unity-icon]: http://splendidabacus.com/posts/2015/03/spacemacs-unity-icon/
[icon-mac-instructions]: http://www.idownloadblog.com/2014/07/16/how-to-change-app-icon-mac/
[icon-repository]: https://github.com/nashamri/spacemacs-logo
[Stack Exchange]: http://emacs.stackexchange.com/questions/tagged/spacemacs
[Reddit]: https://www.reddit.com/r/spacemacs
[quote01]: https://gitter.im/syl20bnr/spacemacs?at=568e627a0cdaaa62045a7df6
[quote02]: https://gitter.im/syl20bnr/spacemacs?at=5768456c6577f032450cfedb
[build_source]: https://www.gnu.org/software/emacs/manual/html_node/efaq/Installing-Emacs.html
[Bountysource]: https://salt.bountysource.com/teams/spacemacs
[Source Code Pro]: https://github.com/adobe-fonts/source-code-pro

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -182,8 +180,8 @@ section in [Documentation][DOCUMENTATION.org].
 
 ### Windows
 
-Good quality builds can be found [on this page][emacs-for-windows]. It is
-recommended to install the most stable build.
+You can download good quality builds from the [emacs-w64 project][emacs-for-windows].
+It is recommended to install the [most stable build][emacs-for-windows-stable].
```
Comment (cpaulik) [inline README.md]: Shouldn't it be `the most recent stable build`?

Comment (hghwng) [inline README.md]: Thanks for the review. It's fixed now.

**Comment by hghwng** [comment]
Commits are squashed to one now.

**Comment by TheBB** [comment]
Thanks! Cherry-picked in develop. You can safely delete your branch.

**Pull request closed without merging**